src/Admin/AbstractAdmin.php

```diff
@@ -2063,7 +2063,26 @@ public function setCurrentChild($currentChild)
         $this->currentChild = $currentChild;
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/admin-bundle 3.x, to be removed in 4.0
+     */
     public function getCurrentChild()
+    {
+        @trigger_error(
+            sprintf(
+                'The %s() method is deprecated since version 3.x and will be removed in 4.0. Use %s::isCurrentChild() instead.',
+                __METHOD__,
+                __CLASS__
+            ),
+            E_USER_DEPRECATED
+        );
+
+        return $this->currentChild;
+    }
+
+    public function isCurrentChild(): bool
     {
```

Comment:
```suggestion
    public function isCurrentChild(bool $newValue = null): bool
    {
        if (null !== newValue) {
            $this->currentChild = $newValue;
        }
```

Nice PR. Can we go even farder and make some change like:
- rename property `currentChild` to `isCurrent`or `isCurrentChild`

Reply:
I don't think it's a good idea to mix setter and getter.
I think it's better to keep both `setCurrentChild` and `isCurrentChild`.